pages/dkp/kommander/1.4/release-notes/index.md

```diff
@@ -21,6 +21,37 @@ To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [ins
 # Release Summary
 Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
 
+## New features
+
+## Workspace Permissions Updates
+
+In previous Kommander versions, Workspace roles were not automatically propagated to Project roles. A user with edit permissions for the Workspace did not automatically receive edit permissions to a Project inside the Workspace. Beginning with Kommander version 1.4, roles are propagated from Workspace to Project, and a user with edit permissions on the Workspace will also have edit permissions on the Project.
+
+<p class=“message--important”><strong>IMPORTANT: </strong>This change will propagate to all Projects, including existing Projects.</p>
+
+Refer to the instructions in [Access Control][access_control] to disable this propagation.
+
+## Network Tunneling
+
+Prior to release 1.4, Kommander required bi-directional connectivity between the Kommander management cluster and clusters that are under management. This effectively blocked several customer-specific use cases:
+
+- Kommander does not have direct access to the managed cluster, for example, when the cluster is on a laptop or behind a NAT gateway.
+
+- The managed cluster does not have direct access to Kommander, for example, Kommander is on-premise and the managed cluster is in a public cloud provider environment.
+
+- The managed cluster is behind a firewall, a proxy, or resides in a DMZ.
+
+A new component, `kubetunnel`, provides communication between Kommander and the managed cluster through a tunneling protocol resolving these blocked use cases. The TLS-encrypted tunnel enables you to access the cluster using SSO and to receive alerts, metrics, and kubecost data.
+
+For more information on this capability, see [Attach an Existing Kubernetes Cluster](../clusters/attach-cluster)
+
+## Catalog Workload Certification
+
+D2iQ now certifies workloads for use with Kommander. All workloads that have been tested and certified for successful configuration and provisioning on Konvoy are flagged in Kommander with a certification icon. For more information, see [Project Platform Services](../projects/platform-services/).
+
+## Helm charts
+The Projects tab now shows all of the current Helm Release charts, their chart version, and the names of the clusters. For more information, see [Project Deployments](../projects/project-deployments/).
+
 # Breaking changes
 
```

Comment:
Need to add COPS tickets to fixes below.

Reply:
Here are the ones I am aware of:

COPS-6897: This one was filed by an internal customer and the 'fix' is described in the "Workspace permissions update" above.   Given that this was an internally raised issue, I see no reason to cite the COPS number here.

(COPS-6912) - This one needs to be referenced, as I know some field folks ran into this and so customers probably did as well.  See (D2IQ-75347) for details on this issue, but we don't need to reference that # in the RN. Suggested RN:  "Fixed a problem where sometimes valid licenses were briefly displayed as invalid."

Reply:
Went through your notes and ensured these were documented as you asked.